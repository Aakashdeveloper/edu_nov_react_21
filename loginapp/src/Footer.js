import React from 'react';

const Footer = (props) => {
    return(
        <>
            <hr/>
            <center>
                <h3>&copy; Developer Funnel</h3>
            </center>
        </>
    )
}

export default Footer