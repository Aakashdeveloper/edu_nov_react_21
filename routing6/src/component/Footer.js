import React from 'react';

const Footer = (props) => {
    return(
        <div style={{display:'inline-block',width:'100%'}}>
            <hr/>
            <center>
                <h3>&copy; Developer Funnel</h3>
            </center>
        </div>
    )
}

export default Footer