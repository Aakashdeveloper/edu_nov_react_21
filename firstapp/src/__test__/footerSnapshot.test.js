import React from 'react';
import Footer from '../component/Footer';
import {create} from 'react-test-renderer';

describe('Snapshot Testing',() => {
    test('Testing Footer',() => {
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})