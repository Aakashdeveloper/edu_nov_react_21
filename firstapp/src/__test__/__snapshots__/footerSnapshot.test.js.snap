// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing Testing Footer 1`] = `
Array [
  <hr />,
  <center>
    <h3>
      Â© Developers Funnel 
       
       
    </h3>
  </center>,
]
`;
